src/components/oscal-utils/OSCALRestUtils.js

```diff
@@ -232,7 +232,7 @@ export function createOrUpdateSspControlImplementationImplementedRequirementStat
   performRequest(
     { "implemented-requirement": partialRestImplementedRequirement },
     restMethods.PUT,
-    requestUrl,
+    buildRequestUrl(null, requestUrl, null),
```

Comment:
We could also do:

```ts
`${process.env.REACT_APP_BASE_URL}/${requestUrl}`,
```

Reply:
I kind of like keeping all of that logic in one place within `buildRequestUrl`